src/psyclone/parse/module_manager.py

```diff
@@ -74,54 +74,33 @@ def get(cache_active: Optional[bool] = None,
             cache_path: Optional[str] = None):
         '''Static function that if necessary creates and returns the singleton
         ModuleManager instance.
-
-        :param use_caching: If `True`, a file-based caching of the fparser
-            tree will be used. This can significantly accelerate obtaining
-            a PSyIR from a source file.
-            For parallel builds, parallel race conditions to the cache file
-            can happen, but this shouldn't lead to wrong results. However,
-            that's untested so far.
-
-        :param cache_path: If set, the cache file will be stored in the given
-            path (directory) using a hashsum of the source code to create
-            a unique cache file name. If `None`, a cache file will be created
-            in the same directory as the source file with a new
-            file ending `.psycache`.
-
         '''
         if not ModuleManager._instance:
-            ModuleManager._instance = ModuleManager(cache_active, cache_path)
+            ModuleManager._instance = ModuleManager()
 
         return ModuleManager._instance
 
     # ------------------------------------------------------------------------
-    def __init__(
-            self,
-            cache_active: Optional[bool] = None,
-            cache_path: Optional[str] = None
-    ):
+    def __init__(self):
         """
         Set up the module manager. Module manager is actually a singleton
         and should not be created directly. Use `ModuleManager.get()`
         instead.
 
-        :param cache_active: Whether to use (`True`) or
-            disable (`False`) caching
-        :param cache_path: Path to the cache directory. If `None`, the
-            cache file will be created in the same directory as the source
-            file with a new file ending `.psycache`.
         """
 
         if ModuleManager._instance is not None:
             raise InternalError("You need to use 'ModuleManager.get()' "
                                 "to get the singleton instance.")
 
         # Disable caching by default
-        self._cache_active = (
-            cache_active if cache_active is not None else False)
+        self._cache_active = False
 
         # Path to cache
-        self._cache_path: Optional[str] = cache_path
+        self._cache_path: Optional[str] = None
+
+        # Whether to resolve imports while inspecting another import
+        self._resolve_indirect_imports = False
```

Comment:
Pls extend this with a typehint too as it's useful.

Reply:
Done